src/sum-of-odd.js

```diff
@@ -26,6 +26,9 @@
  */
 function rowSumOddNumbers(row) {
   // write code here
+  if (row < 0) {
+    return 0;
+  }
+  return Math.pow(row, 3);
```

Comment:
we can write it in one line 
```suggestion
  return row =< 0 ? 0 :  Math.pow(row, 3);
```